module/hooks/init.mjs

```diff
@@ -22,6 +22,7 @@ export default function () {
         // record configuration values
         CONFIG.ED4E = ED4E;
         CONFIG.Actor.documentClass = documents.ActorEd;
+        CONFIG.Actor.documentClass = documents.CharacterEd;
```

Comment:
This feels overkill for that one precreate. Also, this Overrides the line before, which we do not want.